packages/phrases/src/locales/ja/translation/admin-console/applications.ts

```diff
@@ -0,0 +1,53 @@
+const applications = {
+  page_title: 'Applications', // UNTRANSLATED
+  title: 'Applications', // UNTRANSLATED
```

Comment:
Why aren't these two translated? Is it expected?

Reply:
yes it's a rebase issue, i'll update them later